### PR TITLE
feat: Add description for all prop in renderPagination

### PR DESCRIPTION
**Pull request by jyash97**

### CODE REVIEW

File: content/docs/result-components/ReactiveList.md [hunk 1]
```diff
@@ -182,6 +182,16 @@ Example uses:
...
+     - **`pages`**: `number`
+        Number of pages to be displayed
+    - **`totalPages`**: `number`
+        Total number of pages found based on current page size
+    - **`currentPage`**: `number`
+        Current page number for which data is being rendered
+    - **`setPage`**: `function`
+        Function use to set the currentPage
+    - **`fragmentName`**: `string`
+        `componentId` of the Result component for setting url params
```
Comment (bietkul) [inline content/docs/result-components/ReactiveList.md]: @jyash97 How that can be used by a user?

**Review by bietkul — CHANGES_REQUESTED** [review]

File: content/docs/result-components/ReactiveList.md (hunk 1 above)
Comment (jyash97) [reply content/docs/result-components/ReactiveList.md]: Using ```<a href={`?${fragmentName}={pageNumber}`} />```

This how we do in our Pagination component

Comment (bietkul) [reply content/docs/result-components/ReactiveList.md]: It is helpful, can you add it too in docs.

Comment (jyash97) [reply content/docs/result-components/ReactiveList.md]: Done

**Review by bietkul — APPROVED** [review]

**Pull request merged**